src/transformStateWithClones.js

```diff
@@ -6,8 +6,43 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let clonedState = { ...state };
```

Comment:
Use `const` instead `let` here.
Rename this variable to something more descriptive.